saritasa_invocations/alembic.py

```diff
@@ -170,3 +170,100 @@ def _get_migration_files_paths(
         for path in pathlib.Path(migrations_folder).glob("*.py")
         if path.name not in ("__init__.py",)
     )
+
+
+@invoke.task
+def load_db_dump(
+    context: invoke.Context,
+    file: str = "",
+    env_file_path: str = "",
+    reset_db: bool = True,
+) -> None:
+    """Reset db and load db dump."""
+    if reset_db:
+        downgrade(context)
+    db.load_db_dump(
+        context,
+        file=file,
+        **_load_local_env_db_settings(context, file=env_file_path),
+    )
+
+
+@invoke.task
+def backup_local_db(
+    context: invoke.Context,
+    file: str = "",
+) -> None:
+    """Back up local db."""
+    db.backup_local_db(
+        context,
+        file=file,
+        **_load_local_env_db_settings(context),
+    )
+
+
+@invoke.task
+def backup_remote_db(
+    context: invoke.Context,
+    file: str = "",
+) -> str:
+    """Make dump of remote db and download it."""
+    settings = _load_remote_env_db_settings(context)
+    db_k8s.create_dump(context, file=file, **settings)
+    return db_k8s.get_dump(context, file=file)
+
+
+@invoke.task
+def load_remote_db(
+    context: invoke.Context,
+    file: str = "",
+) -> None:
+    """Make dump of remote db, download it and apply it."""
+    file = backup_remote_db(context, file=file)
+    load_db_dump(context, file=file)
+
+
+def _load_local_env_db_settings(
+    context: invoke.Context,
+    file: str = "",
+) -> dict[str, str]:
+    """Load local db settings from .env file.
+
+    Requires python-decouple:
+        https://github.com/HBNetwork/python-decouple
+
+    """
+    # decouple could not be installed during project init
+    # so we import decouple this way because it may not be installed
+    # at the project initialization stage
```

Comment:
```suggestion
    # decouple could not be installed during project init
    # so we import decouple this way to avoid import errors
    # during project initialization
```